utils/prompts.ts

```
@@ -54,6 +68,11 @@ export const getChatPrompt = (options: ChatPromptOptions) => dedent`
 
     # Context
     For your information, today is ${getCurrentDateFormatted()}.
+    
+    # User goals
+    ${(options.goals ?? []).length === 0
+    ? 'The user hasn\'t set any goals yet.'
+    : 'Here is the list of user\'s goals:\n' + options.goals!.join('\n')}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)